Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -129,6 +131,31 @@ func writeDocument(at docRef: DocumentReference) {
     }
 }
 
+func writeDocuments(at docRef: DocumentReference, database db: Firestore) {
+  var batch: WriteBatch;
+
+  batch = db.batch();
+  batch.setData(["a" : "b"], forDocument:docRef);
+  batch.setData(["c" : "d"], forDocument:docRef);
+  // commit without completion callback.
+  batch.commit();
+  print("Batch write without completion complete!")
+
+  batch = db.batch();
+  batch.setData(["a" : "b"], forDocument:docRef);
+  batch.setData(["c" : "d"], forDocument:docRef);
+  // commit with completion callback via trailing closure syntax.
+  batch.commit() { error in
+      if let error = error {
```

Comment:
two space indent.

This should have been fixed by running scripts/style.sh; if not there's a bug in that script.

Make sure to configure you Xcode as described in the (internal) getting started document (under "Acquire a Mac and install Xcode").

Reply:
Done, fixed manually. Looks like `style.sh` does not fix `.swift` file.